csharp/private/frameworks/netcoreapp21.BUILD

```diff
@@ -0,0 +1,1702 @@
+# This file was generated by build\src\D2L.Build.BazelGenerator\FrameworkBuildGenerator\GenerateFrameworkBuildFile.cs
+# Don't edit it by hand! You should regenerate this if the .NET package gets updated.
+
+load("@d2l_rules_csharp//csharp:defs.bzl", "import_library")
+
+import_library(
+    name = "Microsoft.CSharp",
+    target_framework = "netcoreapp2.1",
+    refdll = "//:ref/netcoreapp2.1/Microsoft.CSharp.dll",
+    deps = ["//:netstandard"],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "Microsoft.VisualBasic",
+    target_framework = "netcoreapp2.1",
+    refdll = "//:ref/netcoreapp2.1/Microsoft.VisualBasic.dll",
+    deps = ["//:netstandard"],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "Microsoft.Win32.Primitives",
+    target_framework = "netcoreapp2.1",
+    refdll = "//:ref/netcoreapp2.1/Microsoft.Win32.Primitives.dll",
+    deps = ["//:System.Runtime"],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "mscorlib",
+    target_framework = "netcoreapp2.1",
+    refdll = "//:ref/netcoreapp2.1/mscorlib.dll",
+    deps = [

```

Comment:
Haha I wouldn't have guessed how crazy and arbitrary the dependencies would be in the different frameworks. It's surprising to me that mscorlib depends on these :P

Reply:
Or the generator is broken. I didn't check with ilspy or anything.

Reply:
But this one makes sense in that, nothing depends on mscorlib itself. So seems like it was kept around for legacy reasons and just becomes a "standard library" reference should you reference it.